tests/test_unit/test_prerun_transformations/test_transformations.py

```diff
@@ -0,0 +1,633 @@
+from __future__ import annotations
+
+from contextlib import nullcontext
+from dataclasses import dataclass, field
+from datetime import datetime, timedelta, timezone
+from typing import Any, Literal
+
+import pandas as pd
+import pytest
+import pytz
+from cognite.client import CogniteClient
+from cognite.client.data_classes import (
+    Datapoints,
+    TimeSeries,
+)
+from pydantic import ValidationError
+
+from cognite.powerops.prerun_transformations.transformations import (
+    AddConstant,
+    AddFromOffset,
+    AddWaterInTransit,
+    DoNothing,
+    HeightToVolume,
+    MultiplyConstant,
+    MultiplyFromOffset,
+    OneIfTwo,
+    RelativeDatapoint,
+    Round,
+    StaticValues,
+    SumTimeseries,
+    ToBool,
+    ToInt,
+    ZeroIfNotOne,
+    ms_to_datetime_tz_naive,
+)
+
+
+@pytest.mark.parametrize(
+    "input,expected",
+    [
+        (datetime(2022, 1, 1, 12, tzinfo=pytz.timezone("Etc/UTC")), datetime(2022, 1, 1, 12).replace(tzinfo=None)),
+        (datetime(2022, 1, 1, 12, tzinfo=pytz.timezone("Europe/Oslo")), datetime(2022, 1, 1, 12).replace(tzinfo=None)),
```

Comment:
Do we really want these first two to work - from datetime to datetime, and not from int to datetime as the function name **ms**_to_datetime_tz_naive suggests?